pkgs/servers/gemini/stargazer/default.nix

```diff
@@ -6,25 +6,25 @@
 , scdoc
 , Security
 , nixosTests
+, nix-update-script
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "stargazer";
-  version = "1.2.1";
+  version = "1.3.0";
 
   src = fetchFromSourcehut {
     owner = "~zethra";
     repo = "stargazer";
     rev = version;
-    hash = "sha256-pYize+MGChi1GxCNaQsNlHELtsPUvfFZMPl0Q+pOTp0=";
+    hash = "sha256-Qzg9sCdD29FghHMY6obeVHMD0SRvzi9J6MhEUvcQgbE=";
   };
 
-  cargoHash = "sha256-KmVNRVyKD5q4/vWtnHM4nfiGg+uZvRl+l+Zk5hjWg9E=";
+  cargoHash = "sha256-SQzYEFdSUC1znukot3G3WoOsQGlYP7ICXpoy/w9eO8A=";
 
-  doCheck = false; # Uses external testing framework that requires network
-
-  passthru.tests = {
-    basic-functionality = nixosTests.stargazer;
+  passthru = {
+    tests.basic-functionality = nixosTests.stargazer;
+    updateScript = nix-update-script { };
```

Comment:
Is this required ?
I think you can run `nix-update stargazer` even without specifying this.

Reply:
It does, however I added that in hopes it'd allow the nixpkgs-update bot to work with stargazer, which it currently does not.

Reply:
It currently fails because of the obsolete https://aur.archlinux.org/packages/stargazer displayed on https://repology.org/project/stargazer/versions

I've filled a deletion request on AUR.

ref. https://nixpkgs-update-logs.nix-community.org/stargazer/2024-07-21.log

Reply:
Another idea would be to release a v3 😅

Reply:
Ah oki, thx for figuring that out! I can remove that line from the package if you want.

Reply:
I have no strong opinions, I just wanted to understand.